docs/node/hello-world.md

```diff
@@ -21,11 +21,35 @@ Activities are called from Workflows in order to run non-deterministic code.
 
 Activities are just async functions, they run like typical NodeJS code and can be cancelled and report heartbeats.
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+defaultValue="ts"
+values={[
+{ label: 'TypeScript', value: 'ts', },
+{ label: 'JavaScript', value: 'js', },
+]
+}>
+
+<TabItem value="ts">
+
 `src/activities/greeter.ts`
 
 <!--SNIPSTART nodejs-hello-activity {"enable_source_link": false}-->
 <!--SNIPEND-->
 
+</TabItem>
+
+<TabItem value="js">
+
+<!--SNIPSTART nodejs-js-hello-activity {"enable_source_link": false}-->
```

Comment:
We should add the name of the file here to match the typescript version.
(This applies to all JS snippets).